lib/services/presets.go

```diff
@@ -281,8 +365,9 @@ func defaultAllowLabels() map[string]types.RoleConditions {
 func defaultAllowAccessRequestConditions(enterprise bool) map[string]*types.AccessRequestConditions {
 	if enterprise {
 		return map[string]*types.AccessRequestConditions{
-			teleport.PresetAccessRoleName: {
+			teleport.PresetRequesterRoleName: {
 				SearchAsRoles: []string{
+					teleport.PresetAccessRoleName,
 					teleport.PresetGroupAccessRoleName,
 				},
 			},
```

Comment:
I'm not sure about this one. I don't know that we want to grant those roles explicitly, just resources that belong to those roles.

Reply:
I guess that's more of a product/ux question, but I think it will be hard to change our minds on this once we ship it, I won't block it either way but I think it should be given some thought

Reply:
TBH I think we can always relax the role and add more permissions to it (such as requesting roles in addition to resources) if we decide it improves the UX.